README.md

```diff
@@ -821,8 +820,8 @@ The `forUpdate` quotation can be reused for multiple queries.
 The same approach can be used for `RETURNING ID`:
```

Comment:
I think we could remove this example or replace it by another one now that `returning id` is supported by the DSL.